dbms/src/Debug/astToExecutor.cpp

```diff
@@ -1206,7 +1216,8 @@ void Join::columnPrune(std::unordered_set<String> & used_columns)
     {
         if (left_columns.find(s) != left_columns.end())
             left_used_columns.emplace(s);
-        else
+
```

Comment:
fixed a bug in column prune when left columns and right columns have an column with same name.